### ISSUE TITLE
Repeating/Continuous Map

### ISSUE THREAD

**Issue by bacca87** [radioman/greatmaps#29]
Is possible to set the map continuous?

**Comment by radioman** [comment]
not implemented yet

**Comment by tomlingh** [comment]
I think this and #82 are the same problem. 
Is there an estimate for the work/cost involved to fix this?

**Comment by radioman** [comment]
![F34FE186@EC0F9D5C CB54675E](https://user-images.githubusercontent.com/709019/76643241-95996780-654c-11ea-9776-2691c5d85f75.jpg)
![F5886FCD@4FAFFA2B CB54675E](https://user-images.githubusercontent.com/709019/76643304-b06bdc00-654c-11ea-9fc6-c34352a5d718.jpg)

**Comment by YuryKomar** [comment]
I understand that I, probably, will dublicate the question, but, it is passed 6+ years since first issue opened.
Do you have any improvements?

I am designing the Ships Voyage planning application and still awaiting for map looping on scrolling.

Could you give us some hope for this?
So many apps can do this, your Maps are really Great but only one important feature lacks it :(

**Comment by YuryKomar** [comment]
Also I need to make route by sea surface from Japan to Hawaii... But your map ends somwhere in ocean... :( How to show the route to destination with passing DSL.

**Comment by Fi22xError123** [comment]
Any Solution ?

**Comment by YuryKomar** [comment]
> Any Solution ?

No, still looking for solution. It seems to be at least 3 copies of the map sticked together so route can pass from one to another  via DSL in pacific ocean...

How to make it, no idea.
